wpilibc/src/main/native/cpp/Tracer.cpp

```diff
@@ -16,6 +16,24 @@ Tracer::Tracer() {
   ResetTimer();
 }
 
+void Tracer::PublishToNetworkTables(std::string_view topicName) {
+  if (!m_publishNT){
```

Comment:
Can this be a quick-exit / guard clause instead?

In my experience, quick-exits / guard clauses are more common for this type of usage.